src/components/test-element-settings/TestElementSettings.test.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import TestElementSettings from "./TestElementSettings";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ITestElementSettings } from "@/AppSettings";
+import ElementManager from "@/ElementManager";
+
+const TEST_SETTING: ITestElementSettings = {
+  atomic: 26,
+  enabled: true,
+  stats: {
+    right: 0,
+    times: 0,
+    wrong: 0,
+  },
+};
+
+describe("TestElementSettings", () => {
+  beforeAll(() => {
+    ElementManager.loadElements();
+  });
```

Comment:
This is running in the component `const element = ElementManager.getElement(setting.atomic);`
is there a reason the `.loadElements()` needs to be called manually here? If it does it should be happening in each test within their closures

Reply:
Without that `getElement` won't find the given atomic number because the `ElementManager` would be empty. There is no need to add it in each test case since we just need to load the elements once for them to be available. I could mock `ElementManager.getElement` instead if you think it's a better approach.

Reply:
This ElementManager is an interesting thing, so it must be happening in a parent or ancestor, and since its a class its maintaining that state from instantiation then being used by children component as like a side effect or something.

@nobrayner @mpeyper This looks like it will need to be a Context Hook

Reply:
I've followed the same approach as https://github.com/OpenSourceRaidGuild/atom-pwa/pull/54 and moved it to the setupTests file